contracts/contracts/MACI.sol

```diff
@@ -158,16 +163,8 @@ contract MACI is IMACI, DomainObjs, Params, SnarkCommon, Ownable {
 
         // Check that the factory contracts have correct access controls before
         // allowing any functions in MACI to run (via the afterInit modifier)
-        require(
-            pollFactory.owner() == address(this),
-            "MACI: PollFactory owner incorrectly set"
-        );
-
-        // The VkRegistry owner must be the owner of this contract
-        require(
-            vkRegistry.owner() == owner(),
-            "MACI: VkRegistry owner incorrectly set"
-        );
+        if (pollFactory.owner() != address(this)) revert WrongPollOwner();
```

Comment:
in Solidity if a value is not set it would be set as the default value -> in the address case it would be address(0). Now let's evaluate: address(0) != address(this) -> this would evaluate as false -> address zero (which is not set) is the wrong poll owner. 
Given the contracts inherit the Ownable library, once a contract is deployed, the contract owner becomes the address deploying -> it could only be "not set" (which really is set to address(0)) if they were to renounce ownership -> hence stil incorrect owner.